auth/store.go

```diff
@@ -150,6 +150,9 @@ type AuthStore interface {
 
 	// CheckPassword checks a given pair of username and password is correct
 	CheckPassword(username, password string) (uint64, error)
+
+	// IsAuthEnabled checks authentication is enabled or not
+	IsAuthEnabled() bool
```

Comment:
So long as it's unexported it's OK. I'd like to get rid of the in-package usage, but what I have in mind would be another patchset since it'll need some changes to the etcdserver apply stack.